razorpay/client.py

```diff
@@ -80,6 +83,26 @@ def _get_version(self):
             pass
         return version
 
+    def _get_app_details_ua(self):
+        appDetailsUa = ""
+
+        app_details = self._get_app_details()
```

Comment:
_get_app_details fn is not required(it is private by def), if you want to create a public function remove starting _